hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/restore/RestoreUtils.java

```diff
@@ -43,4 +45,21 @@ public static HoodieRestorePlan getRestorePlan(HoodieTableMetaClient metaClient,
     return TimelineMetadataUtils.deserializeAvroMetadata(
         metaClient.getActiveTimeline().readRestoreInfoAsBytes(requested).get(), HoodieRestorePlan.class);
   }
+
+  /**
+   * Get the restore time for a restore instant
+   * @param table          the HoodieTable
+   * @param restoreInstant Instant referring to restore action
+   * @return restore time of the rollback instant
+   * @throws IOException
+   *
+   * */
+  public static String getRestoreTime(HoodieTable table, HoodieInstant restoreInstant) throws IOException {
+    HoodieRestorePlan plan = getRestorePlan(table.getMetaClient(), restoreInstant);
```

Comment:
I am thinking if we can add an entry to HoodieRestorePlan (avro model) to store the savepoint commit timestamp only. that would make life easier.

Reply:
Thats my preference as well but I didn't know if that was something that could be easily done. We should definitely do that if we can.